ViewControllers/FeatureViewController.swift

```diff
@@ -33,20 +33,22 @@ class FeatureViewController: UIViewController {
 				case .success(let eventType):
 					switch eventType {
 						case .onPolling(let evaluations):
-							for eval in evaluations! {
+							print("Event: Received all evaluation flags")
+                            for eval in evaluations! {
 								self.setupViewPreferences(evaluation: eval)
```

Comment:
do we need these print statements in the sdk ?

Reply:
This all in the sample app.